contrib/pyln-spec/bolt1/setup.py

```diff
@@ -0,0 +1,23 @@
+from pyln.spec.bolt1 import __version__, desc
+from setuptools import setup
+import io
+
+with io.open('requirements.txt', encoding='utf-8') as f:
+    requirements = [r for r in f.read().split('\n') if len(r)]
+
+
+def do_setup(boltnum: int, version: str, desc: str):
+    setup(name='pyln-bolt{}'.format(boltnum),
+          version=version,
+          description=desc,
+          url='http://github.com/ElementsProject/lightning',
+          author='Rusty Russell',
+          author_email='rusty@rustcorp.com.au',
+          license='MIT',
+          packages=['pyln.spec.bolt{}'.format(boltnum)],
+          scripts=[],
+          zip_safe=True,
+          install_requires=requirements)
+
+
+do_setup(1, __version__, desc)
```

Comment:
nit: The indirection through a `do_setup` function that we call is unnecessary as far as I can see.

Reply:
I originally had that one level up, and used symlinks.  I would like to restore that, but it had other problems.

Reply:
Ok, we can revisit this later (or not, since this is really nothing more than a nit).